src/app/column-my.tsx

```diff
@@ -112,57 +113,28 @@ export const columnsMy: ColumnDef<EventMy>[] = [
   {
     id: "actions",
     cell: ({ row }) => {
-      const eventId = row.original.id;
-      const eventName = row.original.name;
-      const eventHost = row.original.host;
-      const eventDescription = row.original.description;
-      const eventQuota = row.original.quota;
-      const eventTime = row.original.time;
-      const eventPlace = row.original.place;
-      const eventCategory = row.original.category;
-      return (
-        <div className="text-blue-500 hover:text-blue-800">
-          <Button variant="warning">
-            <Link
-              href={{
-                pathname: `/edit/${eventId}`,
-                query: {
-                  name: eventName,
-                  host: eventHost,
-                  description: eventDescription,
-                  quota: eventQuota,
-                  time: eventTime,
-                  place: eventPlace,
-                  category: eventCategory,
-                },
-              }}
-            >
-              Edit
-            </Link>
-          </Button>
-        </div>
-      );
-    },
-  },
-  {
-    id: "actions",
-    cell: ({ row }) => {
-      const statusCol = row.original.status;
       const eventName = row.original.name;
       return (
         <div>
           <Dialog>
```

Comment:
create another component for dialog, its better separate dialog with column . Component name is `column` so its list of column, and than call component `dialog` .